### ISSUE TITLE
Add support for VictoriaMetrics tsdb

### ISSUE THREAD

**Issue by valyala** [ntop/ntopng#5449]
Ntopng can use InfluxDB as a tsdb. While InfluxDB is a great time series database, it may use big amounts of memory when collecting stats on a big number of network flows. In this case it would be better using [VictoriaMetrics tsdb](https://github.com/VictoriaMetrics/VictoriaMetrics/), which [uses up to 10x less memory than InfluxDB on high cardinality data](https://valyala.medium.com/insert-benchmarks-with-inch-influxdb-vs-victoriametrics-e31a41ae2893). It shouldn't be hard to integrate, since it can [accept data in InfluxDB line protocol](https://docs.victoriametrics.com/#how-to-send-data-from-influxdb-compatible-agents-such-as-telegraf). The stored data can be read either via [export protocol](https://docs.victoriametrics.com/#how-to-export-data-in-json-line-format) or via [MetricsQL query language](https://docs.victoriametrics.com/MetricsQL.html), which is backwards-compatible with PromQL from Prometheus. PromQL is usually easier to use for typical time series queries than InfluxQL or Flux - see [this article](https://valyala.medium.com/promql-tutorial-for-beginners-9ab455142085).

Probably related issue - https://github.com/ntop/ntopng/issues/3705 .

**Comment by simonemainardi** [comment]
Thanks. It seems you are one of the VictoriaMatrics devs. Are you interested in writing this integration? In case, we can support you with guidance and help.

**Comment by simonemainardi** [comment]
Closing for inactivity. Please, comment or give feedback if interested so we can reopen.

**Comment by tiny-pangolin** [comment]
I believe this can be solved by removing or changing the version checking the in influxdb code. This is the error message below I am getting when using ntopng on opnsense and shipping to a victoriametrics instance on my lan with the influxdb integration without authentication. I am not a coder but I am more than happy to spin a victoriametrics instance up or help one of the devs spin one up.  

```
27/Dec/2021 23:19:05 [prefs.lua:100] [influxdb.lua:1684] ERROR Incompatible InfluxDB version: required "1.5.1", found "nil". Please install an updated version from the official site.

27/Dec/2021 23:17:27 [prefs.lua:100] [influxdb.lua:1684] ERROR Incompatible InfluxDB version: required "1.5.1", found "nil". Please install an updated version from the official site.



```

**Comment by tiny-pangolin** [comment]
using the influxdb listener didn't work for the for same reason since it reports using version 1.0 and received no results when using the influxdb_v2 listener. I was able to get it to work by using an nginx reverse proxy and adding ```add_header X-Influxdb-Version 1.8.0;``` to the main config. After adding that reverse proxy everything appeared in grafana. The full conf is below.


```
server {
    listen 8086;
    #server_name optional.example.com;
    add_header X-Influxdb-Version 1.8.0;
    gzip on;
    gzip_vary on;
    gzip_comp_level 4;
    gzip_min_length 256;
    gzip_proxied expired no-cache no-store private no_last_modified no_etag auth;
    gzip_types application/atom+xml application/javascript application/json application/ld+json application/manifest+json application/rss+xml application/vnd.geo+json application/vnd.ms-fontobject application/x-font-ttf application/x-web-app-manifest+json application/xhtml+xml application/xml font/opentype image/bmp image/svg+xml image/x-icon text/cache-manifest text/css text/plain text/vcard text/vnd.rim.location.xloc text/vtt text/x-component text/x-cross-domain-policy;

    location / {
        proxy_pass http://localhost:8428;
        proxy_http_version 1.1;
        proxy_set_header Connection "Keep-Alive";
        proxy_set_header Proxy-Connection "Keep-Alive";
        proxy_redirect off;
    }
}


```

**Comment by valyala** [comment]
@tiny6996 , the next release of VictoriaMetrics will return the `X-Ifnluxdb-Version` response headers - see https://github.com/VictoriaMetrics/VictoriaMetrics/commit/71ef3155c8733a5b8b4560af80fb557a1b64a71a .

**Comment by valyala** [comment]
VictoriaMetrics returns `X-influxdb-Version: 1.8.0` header from `/write` and `/influx/write` APIs starting from [v1.73.1](https://github.com/VictoriaMetrics/VictoriaMetrics/releases).

**Comment by tiny-pangolin** [comment]
confirmed that it is working with victoriametrics and opnsense

**Comment by barnabasbusa** [comment]
@tiny6996 may I ask if you have managed to get it working with authentication, or without it? 
I seem to be getting the following error messages on my pfsense -> ntopng (influx) -> victoriametrics:

/var/db/ntopng/ntopng.log
```
02/Nov/2022 16:48:18 [prefs.lua:100] [influxdb.lua:1529] ERROR: Could not contact the InfluxDB database: 400
```

Currently using ntopng Community v.5.0.220112 (FreeBSD 12.3)

Current config: 
```
Timeseries Driver : InfluxDB 1.x
InfluxDB URL: https://victoriametrics.mydomain/insert/0/influx/write
InfluxDB Database: ""
InfluxDB Authentication: True
Username: <victoriausername>
Password: <victoriapassword>
Timeseries Resolution: 1m
```

**Comment by tiny-pangolin** [comment]
I did not have auth enabled when I tested this I think the issue is that you are specifying the whole url and just http://victoriametrics.mydomain

**Comment by barnabasbusa** [comment]
When I replace the InfluxDB url with: `https://victoriametrics.mydomain` then I get the same error message as you used to get: ```ERROR: Incompatible InfluxDB version: required "1.5.1", found "nil". Please install an updated version from the <a href='https://portal.influxdata.com/downloads'>official site</a>.``` I thought this was patched in v1.73.1

I still see this error message in `victoriametrics/vminsert:v1.79.2-cluster` and `victoriametrics/victoria-metrics:v1.83.0`
